### ISSUE TITLE
unhandled promise rejection

### ISSUE THREAD

**Issue by Diginix** [iobroker-community-adapters/ioBroker.mihome-vacuum#635]
adapter 3.6.0
js-controller: 4.0.23
nodejs 14.20.0

```
2022-08-07 10:06:26.558 - error: mihome-vacuum.0 (2224300) Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch().
2022-08-07 10:06:26.564 - error: mihome-vacuum.0 (2224300) unhandled promise rejection: undefined
2022-08-07 10:06:26.564 - error: mihome-vacuum.0 (2224300) undefined
2022-08-07 10:06:26.626 - error: mihome-vacuum.0 (2224300) Socket Close
2022-08-07 10:06:26.627 - warn: mihome-vacuum.0 (2224300) Terminated (UNCAUGHT_EXCEPTION): Without reason
2022-08-07 10:06:27.243 - error: host.iobroker Caught by controller[0]: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
2022-08-07 10:06:27.245 - error: host.iobroker Caught by controller[1]: MESSAGE TIMEOUT
2022-08-07 10:06:27.245 - error: host.iobroker instance system.adapter.mihome-vacuum.0 terminated with code 6 (UNCAUGHT_EXCEPTION)

```

**Comment by dirkhe** [comment]
Can you reproduce it with eebug log. Here we can not see, where the exection was thown. I guess, it was in a library

**Comment by dirkhe** [comment]
PLease test with Version https://github.com/iobroker-community-adapters/ioBroker.mihome-vacuum/tree/620-feature-request-queue-shall-contain-fan_power-and-water_box_mode-for-each-command-given, there it is fixed

**Comment by dirkhe** [comment]
Implemented in 3.7.0
